README.md

```diff
@@ -201,6 +201,19 @@ The `"key"` must match a translation value in the `menu` scope (e.g. `menu.more-
 
 A "*Deploy*" menu entry will be set, first, if any custom menu items are included.
 
+### Provisioning bars
+
+The provisioning bars used in the deployment show the progress of each of the individuals provisioning. The provisioning progress is based in the terraform output (that includes salt output) and it looks for certain patterns not know which is the current state of the execution. Certain patters are hardcoded as they belong natively to terraform and salt. Others can be configured in the `customization.json` file. The next example shows how it can be configured. All of the entries must be configured, and they must be populated with regex patterns. This patterns are used to look for specific text in the terraform output to know if there is a state change.
```

Comment:
and it looks for certain patterns not know which is the current state of the execution

should be 

 and it looks for certain patterns **to** know which is the current state of the execution